documentstore_migracao/export/sps_package.py

```diff
@@ -384,7 +400,12 @@ def transform_body(self):
             _, obj_html_body = convert.deploy(txt_body)
 
             # sobrecreve o html escapado anterior pelo novo xml tratado
-            body.getparent().replace(body, obj_html_body.find("body"))
+            if obj_html_body.tag != "body":
+                obj_html_body = obj_html_body.find("body")
+            if obj_html_body is None:
```

Comment:
se o `obj_html_body` for `None` na linha `405` a checagem na linha `403`(`obj_html_body.tag`) vai quebrar?

Reply:
@joffilyfe , essa checagem é para tratar o problema de que em alguns casos a tag raiz não é o Body e o Pipeline de conversão nunca retorna um objeto `None`, mas em alguns caso, os artigos não tem a tag body, e ai `.find()` retorna `None`, por isso que coloquei essa checagem,